dotCMS/src/main/java/com/dotcms/visitor/domain/Visitor.java

```diff
@@ -111,7 +111,7 @@ public Geolocation getGeo() {
     }
     
     public Geolocation getGeo(String ipAddress) {
-        return GeoIp2CityDbUtil.getInstance().getGeolocation(ipAddress);
+        return GeoIp2CityDbUtil.getInstance().getGeolocation("189.209.114.118");
```

Comment:
Codacy found an issue: [Do not hard code the IP address ](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6028986)